lib/pbio/drv/usb/usb_stm32.c

```diff
@@ -10,8 +10,10 @@
 #include <stdbool.h>
 
 #include <contiki.h>
+#include <contiki-lib.h>
```

Comment:
I'll look into this

Reply:
Done!

Just out of curiosity, why was this copied into the repo instead of added as a submodule? How would you feel about me replacing the copy with a submodule?

Reply:
STM32 not being available makes sense. I was referring to `lib/lwrb`.

Aside from being somewhat easier to integrate updates, another advantage of submodules is that you can generate SBOMs (software bill of materials).

I'll submit a new patch that replaces it with a submodule and we'll see how it looks.

Reply:
Well, having looked at this again, I'm not sure we even need a ring buffer here. The Pybricks Profile is message-based rather than stream-based, so I don't think there is going to be any need to put anything in a ring buffer.

Reply:
OK, I'll go ahead and resolve this conversation then and revisit in the future if needed.